### ISSUE TITLE
JAWS announces blank on changing the value by using arrow keys for the spin button

### ISSUE THREAD

**Issue by VaniDivya** [FreedomScientific/VFO-standards-support#890]
Module: APG Patterns
Page name: Date picker Spin button

Page URL: https://www.w3.org/WAI/ARIA/apg/patterns/spinbutton/examples/datepicker-spinbuttons/

Component: Edit spin box

Issue description: When user navigated to Edit spin box and modify the value by using arrow keys, screen reader announces blank before it announces the modified value in it.

Recommended fix: Ensure that screen reader should not announce blank on selecting a value inside spin box using arrow keys.

User impact: If screen reader announces "blank" when selecting a value, it can be confusing and frustrating for users who rely on screen readers to navigate and interact with the control. This can be particularly challenging for users who are blind or have low vision, as they may have difficulty understanding the purpose of the blank announcement.

WCAG Mapping: 1.3.1 Info and Relationships

Level of Compliance: A

Severity: Low

Test tool: JAWS_Firefox
Browser: Firefox

**Comment by JAWS-test** [comment]
I have tested with the latest Chrome and with JAWS 2025 and the problem does not occur. It would be helpful if you could write which JAWS and Firefox version you tested with

**Comment by jhartleytpgi** [comment]
Have tested this with Firefox 137.0.2(64Bit). The reported behavior of announcing "blank" does not occur with Firefox 137.0.2, Chrome Version 135.0.7049.96 (Official Build) (64-bit), and JAWS 2025.2503.39.
